src/main/java/com/paskar/email/application/repositiory/hibernate/CustomRepositoryForProject.java

```diff
@@ -0,0 +1,15 @@
+package com.paskar.email.application.repositiory.hibernate;
+
+import com.paskar.email.application.model.Email;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
```

Comment:
It is not Spring component, it is just an interface.
How do you expect this object will be created without constructor?